### PR TITLE
Chart.2

### PR DESCRIPTION
**Pull request by olu144**
Add chart to display population of a fictional zoo

### CODE REVIEW

**Review by dborkan — COMMENTED** [review]
The `drawChart2` function looks good, you just might want to make sure this code still matches what you want in your master branch, since I think in PR #16  (and maybe others), you've updated the code to call `google.charts.load` only once, like `google.charts.load('current', {'packages':['corechart', 'line']});`

**Comment by olu144** [comment]
I have modified the loadChartsApi function to match what I want in my master branch.

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -93,14 +93,42 @@ function drawChart() {
...
-  google.charts.load('current', {'packages':['corechart']});
+  google.charts.load('current', {'packages':['corechart', 'line']});
   google.charts.setOnLoadCallback(drawChart);
+  google.charts.setOnLoadCallback(drawChart2);
 }
+
+/*function loadChartsApi2() {
+  google.charts.load('current', {'packages':['line']});
+  google.charts.setOnLoadCallback(drawChart2);
+}*/
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js]: nit: this commented out code should be removed

**Review by dborkan — APPROVED** [review]
👍 code added for this PR looks good, you'll just want to double check when you merge that there are no conflicts with the other recent PRs

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/webapp/script.js]: Addressed in "remove commented code in script.js"

**Pull request merged**